### ISSUE TITLE
Unable to start vagrant mantl with latest code

### ISSUE THREAD

**Issue by subbu05** [mantl/mantl#1829]
git clone latest code from github
Ran below command 
vagrant up

```
No vagrant-config.yml found, using defaults
Bringing machine 'worker-001' up with 'virtualbox' provider...
Bringing machine 'kubeworker-001' up with 'virtualbox' provider...
Bringing machine 'edge-001' up with 'virtualbox' provider...
Bringing machine 'control-01' up with 'virtualbox' provider...
==> worker-001: Checking if box 'CiscoCloud/mantl' is up to date...
==> worker-001: Machine already provisioned. Run `vagrant provision` or use the `--provision`
==> worker-001: flag to force provisioning. Provisioners marked to run always will still run.
==> kubeworker-001: Checking if box 'CiscoCloud/mantl' is up to date...
==> edge-001: Importing base box 'CiscoCloud/mantl'...
==> edge-001: Matching MAC address for NAT networking...
==> edge-001: Checking if box 'CiscoCloud/mantl' is up to date...
==> edge-001: Setting the name of the VM: mantl_edge-001_1473273659613_59687
==> edge-001: Fixed port collision for 22 => 2222. Now on port 2201.
==> edge-001: Clearing any previously set network interfaces...
==> edge-001: Preparing network interfaces based on configuration...
    edge-001: Adapter 1: nat
    edge-001: Adapter 2: hostonly
==> edge-001: Forwarding ports...
    edge-001: 22 (guest) => 2201 (host) (adapter 1)
==> edge-001: Running 'pre-boot' VM customizations...
==> edge-001: Booting VM...
==> edge-001: Waiting for machine to boot. This may take a few minutes...
    edge-001: SSH address: 127.0.0.1:2201
    edge-001: SSH username: vagrant
    edge-001: SSH auth method: private key
    edge-001: Warning: Remote connection disconnect. Retrying...
    edge-001: Warning: Remote connection disconnect. Retrying...
    edge-001: Warning: Remote connection disconnect. Retrying...
    edge-001: 
    edge-001: Vagrant insecure key detected. Vagrant will automatically replace
    edge-001: this with a newly generated keypair for better security.
    edge-001: 
    edge-001: Inserting generated public key within guest...
    edge-001: Removing insecure key from the guest if it's present...
    edge-001: Key inserted! Disconnecting and reconnecting using new SSH key...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
    edge-001: Warning: Authentication failure. Retrying...
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.
```

**Comment by msowka-ninja** [comment]
This is a bug in vagrant:
https://github.com/mitchellh/vagrant/issues/7610

**Comment by zuaresc** [comment]
I have gotten past this by adding `config.ssh.insert_key = false` below `config.vm.box = "CiscoCloud/mantl"` in the Vagrantfile.

**Comment by thebillkidy** [comment]
I fixed this by installing vagrant 1.8.5, the latest version is bugged but this should be fixed in 1.8.7 which is being released later this month.

**Comment by ekka21** [comment]
I had the same problem while using macOS Sierra v10.12.1, Vagrant v1.8.5, and VirtualBox v5.1.6. 
.
..
...
I confirm that reinstalling Vagrant to latest version(1.9.0) is fixed the problem. :)

**Comment by Cryptophobia** [comment]
@subbu05 @ekka21 @thebillkidy : Sounds like there was a workaround merged to fix this ( 7f6bbca ) and it is also resolved when using a later version of Vagrant. So I am closing this issue.
